solidstart-tanstackquery-tailwind-modules/src/components/FilterDropDown/FilterDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import { createSignal, Show, splitProps, For, Switch, Match } from 'solid-js';
+import { CaretIcon, CloseIcon, CorrectIcon } from '../Icons';
+import { clickOutside } from '../../utils/onclickOutside';
+import styles from '../RepoFilter/RepoFilter.module.css';
+
+const FilterDropdown = (props) => {
+  const [local, others] = splitProps(props, [
+    'name',
+    'title',
+    'items',
+    'selected',
+    'selectOption',
+  ]);
+  const [showOptions, setShowOptions] = createSignal(false);
+  const toggleOption = () => setShowOptions(!showOptions());
+  const _clickOutside = clickOutside;
+
+  return (
+    <div
+      data-testid="filter-dropdown"
+      class={styles.dropDownContainer}
```

Comment:
please add   // @ts-ignore after this line in order not to have ts error in the code